src/rnn/rnn_util.cpp

```diff
@@ -311,6 +311,10 @@ void RNNTensorBaseLayoutConverter::ChangeTensorGPUDataPadding(
         const std::vector<size_t> packed_stride =
             get_packed_stride(copy_size, tensor_desc.GetLayoutVector());
 
+        // Nothing to copy, avoiding error with zero lens in TensorDescriptor
+        if(!std::all_of(copy_size.cbegin(), copy_size.cend(), [](size_t x) { return x > 0; }))
+            continue;
```

Comment:
`std::all_of` returns true if the range is empty, https://en.cppreference.com/w/cpp/algorithm/all_any_none_of

Reply:
[resolved] @shurale-nkn Now I see.